thrift/jaeger.thrift

```diff
@@ -54,14 +54,15 @@ struct SpanRef {
 struct Span {
   1:  required i64           traceIdLow   # the last 64 bits of a traceID
   2:  required i64           traceIdHigh  # the first 64 bits of a traceID
```

Comment:
* in the comments, replace last/first with "least/most significant"
* add to High comment: "; 0 when only 64bit IDs are used"